content/guides/references/migration-guide.md

```diff
@@ -2,14 +2,256 @@
+what generally available implies [here](link_to_post) and the improvements
+[here](annoucement_blog_post).
+
+There are some minor breaking changes. Most projects should be able to migrate
+without any code modifications.
+
+#### Changes to Mounting Options
+
+Each major libary we support has a `mount` function with two arguments:
+
+1. The component
+2. Mounting Options
+
+Mounting options previously had several properties that are now removed:
+
+- cssFile, cssFiles
+- style, styles
+- stylesheet, stylesheets
+
+Read more about the rationale [here](link_to_explainer). We recommend writing
+test-specific styles in a separate `css` file you import in your test, or in
+your `supportFile`.
+
+#### Before (Cypress 10)
+
+```jsx
+import { mount } from ‘cypress/react’
+import { Card } from ‘./Card’
+
+it(‘renders some content’, () => {
+  cy.mount(<Card title=”title” />, {
+    styles: `
+      .card { width: 100px; }
+    `,
+    stylesheets: ['https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css']
+  })
+})
+```
+
+#### After (Cypress 11)
+
+```js
+/** style.css */
+@import "https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css";
+.card { width: 100px }
+
+/** Card.cy.jsx */
+import { mount } from ‘cypress/react’
+import { Card } from ‘./Card’
+import ‘./styles.css’ // contains CDN link and custom styling.
+
+it('renders some content', () => {
+  cy.mount(<Card title=”title” />)
+})
+```
+
+### React - `mountHook` Removed
+
+`mountHook` from `cypress/react` has been removed. Replace it with `mount` and a
+component.
+
+Consider the following `useCounter` hook:
+
+```js
+import { useState, useCallback } from 'react'
+
+function useCounter() {
+  const [count, setCount] = useState(0)
+  const increment = useCallback(() => setCount((x) => x + 1), [])
+
+  return { count, increment }
+}
+```
+
+#### Before - Cypress 10 and `mountHook`
+
+```js
+import { mountHook } from 'cypress/react'
+import { useCounter } from ‘./useCounter’
+
+it('increments the count', () => {
+  mountHook(() => useCounter()).then((result) => {
+    expect(result.current.count).to.equal(0)
+    result.current.increment()
+    expect(result.current.count).to.equal(1)
+    result.current.increment()
+    expect(result.current.count).to.equal(2)
+  })
+})
+```
+
+#### After - Cypress 10 and `mount`
+
+```js
+import { useCounter } from ‘./useCounter’
+
+it('increments the count', () => {
+  function Counter () {
+    const { count, increment } = useCounter()
+    return (
+      <>
+        <h1 name="count">Count is {{ count }}</h1>
+        <button onClick={increment}>Increment</button>
+      </>
+    )
+  }
+
+  cy.mount(<Counter />).then(() => {
+    cy.get('[name="count"]').should('contain', 0)
+      .get('button').click()
+      .get('[name="count"]').should('contain', 1)
+  })
+})
+```
+
+### React - `unmount` Removed
+
+`unmount` from `cypress/react` has been removed. We recommend using the API
+React provides for unmount components.
```

Comment:
Should there be a link to the referenced React API?

Reply:
Done